code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -2188,7 +2188,7 @@ datum
 						//make it obvious that you are about to die horribly
 						M.addOverlayComposition(/datum/overlayComposition/flockmindcircuit)
 						// oh no
-						if(probmult(1)) // i hate you all, players
+						if(probmult(max(2, (src.volume - gib_threshold)/5))) // i hate you more, players
```

Comment:
is `src.volume` right? we use `holder.get_reagent_amount(src.id) ` above. Also no need for the max since volume is necessarily greater than `gib_threshold` in this if block

Reply:
`src.volume` seems to be fine, asked pali about it, and the max is needed since we're dividing by 5 and want it to always be at least 2 (eg `(202-200)/5 = 0.4`)